### PR TITLE
Update the catalog Scheduled Changes pages

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request updates the catalog Scheduled Changes pages

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/catalog/category-scheduled-changes.html?itm_source=merchdocs&itm_medium=search_page&itm_campaign=federated_search&itm_term=Sched
https://docs.magento.com/user-guide/catalog/product-scheduled-design-update.html?itm_source=merchdocs&itm_medium=search_page&itm_campaign=federated_search&itm_term=Sched

### CODE REVIEW

File: src/catalog/category-scheduled-changes.md [hunk 1]
```diff
@@ -7,3 +7,21 @@ Category updates can be applied on schedule, and grouped with other content chan
...
 ![]({% link images/images-ee/category-scheduled-changes.png %}){: .zoom}
 _Scheduled Changes_
+
+## Schedule an update to a category
+
+1. Go to the **Categories** page and open a category in edit mode.
+
+1. In the _Scheduled Changes_ box at the top of the page, click <span class="btn">Schedule New Update</span>.
+
+1. With the **Save as a New Update** option selected, Set the basic parameters for the update:
```
Comment (leslietilling) [inline src/catalog/category-scheduled-changes.md:17]:
```suggestion
1. With the **Save as a New Update** option selected, set the basic parameters for the update:
```

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**